### DISCUSSION TITLE
Provide Gradle Plugin

### DISCUSSION THREAD

**Discussion by TSampley** [mockative/mockative#86] [Ideas]
# User Story
As a Gradle Project Developer, I would like to apply dependencies through a gradle plugin, to simplify setup

## Context
The project setup is already minimal and easy to copy-and-paste, but it assumes conventions that may change.
1. A plugin that manages this configuration behind the scenes would be a slight improvement for initial setup, and–
2. it may be a major life-saver in the future if it were extended to read the KMP plugin version to modify its setup (assuming some future breaking change to conventions)

This was originally written to be a ticket, but I thought a discussion might be more productive; let me know if you'd like me to convert it back to a ticket

**Comment by glureau** [comment]
Mockative3 does offer a plugin now 🎉

**Reply by riQQ** [reply]
See also #96
